### ISSUE TITLE
Genotyping variants

### ISSUE THREAD

**Issue by Akazhiel** [friend1ws/nanomonsv#13]
Greetings!

Do you have any plans on implementing the genotyping of the variants? Or if you recommend tool that could genotype the VCF output of nanomonsv.

Cheers!

**Comment by friend1ws** [comment]
Hi, could you explain more about the "genotyping the variant" in this context?

**Comment by Akazhiel** [comment]
Hello!

Yes of course. In the VCF it's reported the number of reads that are checked in the normal and tumor and the number of reads that support the variant in normal and tumor. I was asking if the variant could be genotyped in each sample like other SV callers do and note if it's homozygous 0/0 or homozygous 1/1 or heterozygous 0/1. And maybe change the FORMAT field to a more standard nomenclature with `GT:DR:DV`, where each of the keys correspond to:

```
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=DR,Number=1,Type=Integer,Description="# High-quality reference reads">
##FORMAT=<ID=DV,Number=1,Type=Integer,Description="# High-quality variant reads">
```

**Comment by skeremaydin** [comment]
@Akazhiel you are right! The deficiency of this tool is hindering downstream bioinformatics processes, to the extent that annotation is not even being performed.
